Fish-Registration/Views/Shared/_Layout.cshtml

```diff
@@ -73,6 +80,20 @@
         <script src="~/js/site.min.js" asp-append-version="true"></script>
     </environment>
 
+    <!-- Start review stars -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```

Comment:
In reference to my comment below for the jquery-bar-rating library, I would remove this reference to jquery itself though. See [this](https://github.com/thejohnjensen/fisher-registration/blob/master/Fish-Registration/Views/Shared/_Layout.cshtml#L56). `_Layout.cshtml` is included as part of every view file so jquery is loaded as part of that automatically. Also, the version of jquery it loads is different from this version.